src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -4567,6 +4575,7 @@ bool StorageReplicatedMergeTree::optimize(
 
     auto try_assign_merge = [&](const String & partition_id) -> bool
     {
+        LOG_INFO(log, "## -- try_assign_merge --\n");
```

Comment:
Please make this LOG_TRACE

Reply:
This log (and next LOG_INFO in this file) is part of the remaining logs we planned to remove before merge.